mysqld_exporter.go

```diff
@@ -45,6 +45,14 @@ var (
 		"collect.perf_schema.indexiowaitstime", true,
 		"Collect time metrics from performance_schema.table_io_waits_summary_by_index_usage",
 	)
+	perfEventsStatements = flag.Bool(
+		"collect.perf_schema.eventsstatements", true,
```

Comment:
Yea, I have been waffling on this.  I'm going to say yes, off by default for first release.